benchmarks/benchmark_shared.c

```diff
@@ -129,6 +129,9 @@ int bandwidth(void)
 	if (ret)
 		return ret;
 
+	struct fi_context *ctx_arr = 
+		calloc(opts.window_size, sizeof(struct fi_context));
+
```

Comment:
Can the context allocation be done in some other initialization related code area?